app/build.gradle

```diff
@@ -7,6 +7,7 @@ apply from: './jacoco.gradle'
 android {
     lintOptions {
         warning 'InvalidPackage'
+        abortOnError false
```

Comment:
lint was failing due to unmatch of palette version used in glidePalette library and support library version used in the app.